### PR TITLE
Update Readme docs

### LINKED ISSUE #432: Use clusterctl generate cluster command
/kind feature
/area provider/ibmcloud

**Describe the solution you'd like**
Currently `clusterctl config cluster` is used in the template [readme](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/tree/main/templates) doc, which is deprecated and throws the following message while parsing the template:

```shell
Command "cluster" is deprecated, use `clusterctl generate cluster` instead
```

Needs to be replaced with the `clusterctl generate cluster` command

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot-Parab <prajyot.parab2@ibm.com>

**What this PR does / why we need it**:
1. Update readme docs
2. Add PowerVS specific cluster deployment doc

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #432 

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by mkumatag** [comment]
@Prajyot-Parab please fix the cla,

/ok-to-test

**Comment by Prajyot-Parab** [comment]
I signed it

File: templates/ibmcloud-powervs-doc.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+### Provision local boostrap management cluster:
```
Comment (mkumatag) [inline templates/ibmcloud-powervs-doc.md:1]: we have started looking at hosting a doc for the project and the layout can be found here: https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/tree/main/docs/book/src,

and this content can go there and here is how it looks like post-deployment: cluster-api-aws.sigs.k8s.io/ and also you can refer some of the content from the other provider

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: templates/ibmcloud-powervs-doc.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+4. Provision workload cluster in IBM Cloud PowerVS
+
+    You can use clusterctl to render the yaml through templates.
+
+    **Note:** the `IBMPOWERVS_IMAGE_ID` value below should reflect the ID of the custom qcow2 image
+
+    ```console
+    IBMPOWERVS_SSHKEY_NAME="my-pub-key" \
+    IBMPOWERVS_VIP="192.168.151.22" \
+    IBMPOWERVS_VIP_EXTERNAL="158.175.162.22" \
```
Comment (Amulyam24) [inline templates/ibmcloud-powervs-doc.md:71]: Might be better to add how the assigned VIP is generated here

Comment (wentao-zh) [reply templates/ibmcloud-powervs-doc.md:71]: Agree with @Amulyam24 . If the doc link of PowerVS on IBM Cloud can be added as well it should be helpful.

**Comment by Prajyot-Parab** [comment]
I signed it

**Comment by Prajyot-Parab** [comment]
I signed it

**Review by Amulyam24 — APPROVED** [review]
/lgtm

File: docs/book/src/topics/powervs/creating-a-cluster.md [hunk 3]
```diff
@@ -1 +1,181 @@
...
+
+    ```console
+    ~ kubectl get machinedeployments
+    NAME              PHASE       REPLICAS   READY   UPDATED   UNAVAILABLE
+    ibm-powervs-1-md-0    Running     2          2       2
+
+    ~ kubectl scale machinedeployment ibm-powervs-1-md-0 --replicas 3
+    ```
+
+    Increase / decrease the `replicas: 2` count in the spec section to see the machine replicas reconciled within the workload cluster.
```
Comment (mkumatag) [inline docs/book/src/topics/powervs/creating-a-cluster.md:181]: This isn't required, because this is something based on the user's choice.

**Review by mkumatag — CHANGES_REQUESTED** [review]
more instructions required which talks about how to create powervs service instance and the public network as well, that can go into the prereq.

**Review by mkumatag — APPROVED** [review]
decent content for first cut, will enhance it further.
/lgtm

**Pull request merged**